common/changes/@itwin/core-frontend/geo-mapfeatureInfo_fixes_2023-09-13-13-30.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Added parameter to specify pixel tolerance of Viewport.getMapFeatureInfo",
+      "type": "none"
```

Comment:
Since you are augmenting a public api, both of these changes should be minor.

Reply:
Do I need to edit that manually?  `rush change` didn't offer me any option.

Reply:
we never specify the change type since the repo is in lockstep.
we just need to be careful not to bp this into a patch

Reply:
Ah right... thanks.